recipes/tramway/meta.yaml

```
@@ -0,0 +1,61 @@
+
+{% set name = "tramway" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DecBayComp/TRamWAy/archive/v{{ version }}.tar.gz
+  sha256: 6e26acc4280dfb6d93f1d5009009927dd1fa2e63f18a2c4e160b9a1e40534bc6
+
+build:
+  skip: True  # [py<36]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

`conda build` works fine with `noarch: python` + `python >=3.6`, indeed, but `conda-forge-linter` rejects any combination of `noarch` and selectors such as `[not win]`, as can be seen in the trail of attempts above.
I also had to remove tests with a `[not win]` selector because of `conda-forge-linter`, whereas `conda build` ran alright with it.
There is still one such selector in the recipe.

---

Ah my bad, I must have missed that. In that case this is fine.